docs/guides/caching.md

```diff
@@ -0,0 +1,171 @@
+---
+title: "Caching in Cortex"
+linkTitle: "Caching in Cortex"
+weight: 1
+slug: caching
+---
+
+# Caching in Cortex
+
+Correctly configured caching is important for a production-ready Cortex cluster.
+Cortex has many opportunities for using caching to accelerate queries and reduce cost.  Cortex can use a cache for:
+* Individual chunks
```

Comment:
We should make more clear what applies to the chunks storage only and what to any chunk engine.

Ie.
```
Chunks storage engine:

* Individual chunks
* Index lookups for one label on one day
* Reducing duplication of writes

Any storage engine:

* The results of a whole query
```

I would also improve the doc below (when you talk about the caching opportunities) to re-iterate when they applies to the chunks storage engine only.